debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerPartition.java

```diff
@@ -82,7 +82,7 @@ static class Provider implements Partition.Provider<SqlServerPartition> {
         public Set<SqlServerPartition> getPartitions() {
             String serverName = connectorConfig.getLogicalName();
 
-            return Arrays.stream(taskConfig.getString(DATABASE_NAMES.name()).split(","))
```

Comment:
This makes the `taskConfig` property and the constructor parameter no longer used anywhere.